tests/study_group/test_study_group.py

```diff
@@ -56,3 +56,8 @@ def test_field_over_char_limit(self, persist_group, over_char_limit):
         persist_group.field = over_char_limit
         with pytest.raises(ValidationError):
             persist_group.save()
+
+    def test_join_get_members_count(self, persist_group):
+        for i in range(1, persist_group.capacity):
+            persist_group.join_group(User.objects.create(username=f'user{i}', password='pass'))
```

Comment:
Same as above

Reply:
The reason I don’t use fixture here is because the number of users to be created is not fixed but dependent on the group capacity